keyboards/crkbd/keymaps/manna-harbour/keymap.c

```diff
@@ -0,0 +1,181 @@
+// https://github.com/manna-harbour/qmk_firmware/blob/crkbd/keyboards/crkbd/keymaps/manna-harbour/readme.org
+
+#include QMK_KEYBOARD_H
+
+#if defined MH_OLED_IMAGE_FILE && defined OLED_DRIVER_ENABLE
+#include "mh_image.h"
+#endif
+
+#if defined CONSOLE_ENABLE
+#include <print.h>
+#endif
+
+#if defined PS2_MOUSE_ENABLE
+#include "ps2_mouse.h"
+#endif
+
+#if defined MOUSEKEY_ENABLE
+#include "mousekey.h"
+#endif
+
+#if defined SSD1306OLED
+#include "ssd1306.h"
+#endif
```

Comment:
Ideally, the `SSD1306OLED` code shouldn't be used, at all. Instead, the oled driver feature should be used in its placen

Reply:
```suggestion
```